package.json

```
@@ -443,7 +443,7 @@
     "@elastic/eslint-plugin-eui": "0.0.2",
     "@elastic/github-checks-reporter": "0.0.20b3",
     "@elastic/makelogs": "^6.1.1",
-    "@elastic/synthetics": "^1.0.0-beta.12",
+    "@elastic/synthetics": "~1.0.0-beta.13",
```

```suggestion
    "@elastic/synthetics": "~1.0.0-beta.39",
```

this should also work with node which is compatible with 7.17